It seems to fail on ofborg
```
machine: must succeed: mkdir /var/lxd-pool
machine # [   18.636334] lxd[627]: t=2020-09-09T17:36:49+0000 lvl=warn msg=" - Couldn't find the CGroup blkio.weight, I/O weight limits will be ignored"
(0.05 seconds)
machine: must succeed: cat /nix/store/64l16k960yab010xv69spwrqbvmjam72-config.yaml | lxd init --preseed
machine # Error: Failed to connect to local LXD: Get "http://unix.socket/1.0": dial unix /var/lib/lxd/unix.socket: connect: no such file or directory
machine: output:
error:
Traceback (most recent call last):
  File "/nix/store/kcy6z1kbf4qd6jvv6y9c9g43sdsxy466-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 898, in run_tests
    exec(tests, globals())
  File "<string>", line 1, in <module>
  File "<string>", line 10, in <module>
  File "/nix/store/kcy6z1kbf4qd6jvv6y9c9g43sdsxy466-nixos-test-driver/bin/.nixos-test-driver-wrapped", line 421, in succeed
    raise Exception(
Exception: command `cat /nix/store/64l16k960yab010xv69spwrqbvmjam72-config.yaml | lxd init --preseed` failed (exit code 1)
cleaning up
```

--- a/nixos/tests/lxd.nix
+++ b/nixos/tests/lxd.nix
@@ -8,12 +8,12 @@ let
   # generally, sufficient for our tests.
   alpine-meta = pkgs.fetchurl {
     url = "https://tarballs.nixos.org/alpine/3.12/lxd.tar.xz";
-    sha256 = "sha256-1tcKaO9lOkvqfmG/7FMbfAEToAuFy2YMewS8ysBKuLA=";
+    hash = "sha256-1tcKaO9lOkvqfmG/7FMbfAEToAuFy2YMewS8ysBKuLA=";
   };
 
   alpine-rootfs = pkgs.fetchurl {
     url = "https://tarballs.nixos.org/alpine/3.12/rootfs.tar.xz";
-    sha256 = "sha256-Tba9sSoaiMtQLY45u7p5DMqXTSDgs/763L/SQp0bkCA=";
+    hash = "sha256-Tba9sSoaiMtQLY45u7p5DMqXTSDgs/763L/SQp0bkCA=";
   };
 
   lxd-config = pkgs.writeText "config.yaml" ''

--- a/nixos/tests/lxd.nix
+++ b/nixos/tests/lxd.nix
@@ -68,6 +68,7 @@ in {
   testScript = ''
     machine.wait_for_unit("sockets.target")
     machine.wait_for_unit("lxd.service")
+    machine.wait_for_file("/var/lib/lxd/unix.socket")
 
     # It takes additional second for lxd to settle
     machine.sleep(1)

